cowprotocol/accounting/rewards/reward_capping_experiments/rewards_per_solver_5930664.sql

```diff
@@ -0,0 +1,14 @@
+with rewards_per_solver as (
+    select
+        solver_name,
+        sum(protocol_fee / 1e18) as protocol_fee,
+        sum(new_protocol_fee / 1e18) as new_protocol_fee,
+        sum(reward / 1e18) as reward,
+        sum(new_reward_proposal / 1e18) as new_reward_proposal,
+        sum(new_reward_fee_cap / 1e18) as new_reward_fee_cap
```

Comment:
should also select `sum(volume)` and count(*) so we can put values into perspective of total batches/volume the solver provides.